src/niclips/figures/multi_view.py

```diff
@@ -33,7 +33,7 @@ def multi_view_frame(
     """Construct a multi view image panel. Returns a PIL Image."""
     check_3d(img)
     check_iso_ras(img)
-    if overlay is not None:
+    if overlay:
```

Comment:
Fine to make this change in most places. But personally I am often afraid to do plain `if x` instead of `if x is not None` since they are not strictly equivalent. E.g. `if x` raises an error if `x` is a numpy array.